camera_system/coordinate_getter.py

```diff
@@ -0,0 +1,81 @@
+"""座標取得GUIツール.
+
+画像を表示し、クリックした座標を保持する
+@author miyashita64 mutotaka0426
+@note 参考: https://techacademy.jp/magazine/51035
+"""
+
+import cv2
+from typing import List
+
+
+class CoordinateGetter:
+    """画像から座標を取得するクラス.
+
+    Attributes:
+        __block_point: ブロック置き場の座標リスト
+        __base_circle: ベースサークルの座標リスト
+    """
+
+    def __init__(self, file_name: str) -> None:
+        """カメラシステムのコンストラクタ.
+
+        Args:
+            file_name (str, optional): 入力画像ファイルのパス.
+        """
+        # メンバを初期化する.
+        self.__block_point = []
+        self.__base_circle = []
+
+        # GUIツールを設定する
+        self.__img = cv2.imread(file_name)
+        self.__window_name = "Coordinate Getter"
+        # 画像を表示する.
+        self.__showImage()

```

Comment:
コンストラクタで、画像表示と座標取得までするのは荷が重い？気がします。

以下のように、コンストラクタでメンバ変数の初期化のみを行えば、少なくともコンストラクタと`block_point`、`base_circle`のテストは可能になるかと思います。

```suggestion
    def __init__(self) -> None:
        """CoordinateGetterのコンストラクタ."""
        # メンバを初期化する.
        self.__block_point = []
        self.__base_circle = []
        self.__window_name = "Coordinate Getter"
```